client/client.go

```diff
@@ -81,6 +82,14 @@ type Client struct {
 	manualOverride bool
 }
 
+func redirectError(req *http.Request, via []*http.Request) error {
+	if via[0].Method == http.MethodGet {
+		return nil
+	}
+
+	return errors.New("Sorry daemon could not process your request. Please make sure you entered correct arguments")
```

Comment:
Could we make this error a package var and simplify the error? The error should be focused on the particular issue that's being checked, then the CLI can check the error and add more context.
Maybe something like `unexpected redirect in response`.

Note that I believe the net/http client will wrap this in a `url.Error`.